### DISCUSSION TITLE
How can I include my test code under the "Files changed" tab

### DISCUSSION THREAD

**Discussion by o-P-o** [community/community#21942] [New to GitHub]
<p>I have successfully issued a pull request, and on my pull request page there is a “Files changed” tab, which includes the edits I made to the file. </p>
<p>In this section I want to also include my test code that I used to test my edits.</p>
<p>How do I do this?</p>

**Comment by mpboom** [comment]
<p>The tab represents all changes that were made in all commits that are part of the PR. Just make sure you commit the changes you made to the files you used to test (and also make sure that those commits are part of the PR) and they will be included. There is no manual way to add files there.</p>

**Comment by o-P-o** [comment]
<p>OK, so once I’ve already issued a PR and it’s up, how can I then create another commit as part of this PR so that my test code can be seen?</p>

**Comment by mpboom** [comment, marked as answer]
<p>Just push more commits towards the branch you want to merge into another branch. The PR will automatically update.</p>

**Comment by o-P-o** [comment]
<p>(I have the test code as a file on my git account.)</p>
